source/tutorials/sample-classifier.rst

```diff
@@ -28,7 +28,22 @@ Supervised learning classifiers predict the categorical metadata classes of unla
    :url: https://data.qiime2.org/2018.8/tutorials/sample-classifier/moving-pictures-table.qza
    :saveas: moving-pictures-table.qza
 
-Next, we will attempt to predict which body site each sample originated from based on its microbial composition.
+Next, we will train and test a classifier that predicts which body site a sample originated from based on its microbial composition. We will do so using the ``classify-samples`` pipeline, which performs a series of steps under the hood:
+
+1. The input samples are randomly split into a ``training`` set and a ``test`` set. The test set is held out until the end of the pipeline, allowing us to test accuracy on a set of samples that was not used for model training. The fraction of test samples to remove is adjusted with the ``--p-test-size`` parameter.
```

Comment:
The word _remove_ doesn't seem right here: _The fraction of test samples to remove is adjusted with the_

How about: _The fraction of input samples to include in the test set is adjusted with the_